cli/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
What do you think leaving this `.editorconfig` file equal to the main `.editorconfig` of the root on *mern-seed*? Or do we need also this? Thanks @matiasbeckerle